docs/manual.xml.head

```diff
@@ -16198,6 +16198,85 @@ bind index &gt; vfolder-window-forward
       </sect2>
     </sect1>
 
+    <sect1 id="pager-read-delay-feature">
+      <title>Pager Read Delay Feature</title>
+      <subtitle>Delay when the pager marks a previewed message as
+      read</subtitle>
+
+      <sect2 id="pager-read-delay-support">
+        <title>Support</title>
+        <para>
+          <emphasis role="bold">Since:</emphasis> NeoMutt 2021-06-??
+        </para>
+        <para>
+          <emphasis role="bold">Dependencies:</emphasis> None
+        </para>
+      </sect2>
+
+      <sect2 id="pager-read-delay-intro">
+        <title>Introduction</title>
+        <para>
+          The <quote>Pager Read Delay</quote> feature adds a new
+          config variable to allow the pager to operate in a preview
+          mode.  A new message is not marked as read merely because
+          the pager opened it, but only after the pager remains on the
+          message for a given length of time.
+        </para>
+      </sect2>
+
+      <sect2 id="pager-read-delay-functions">
+        <title>Functions</title>
+        <para>
+          The <quote>Pager Read Delay</quote> feature adds no new
+          functions NeoMutt.  Existing pager functions for navigating
+          to a different message now check whether to mark a message
+          as read.
+        </para>
+      </sect2>
+
+      <sect2 id="pager-read-delay-variables">
+        <title>Variables</title>
+        <para>
+          The <quote>Pager Read Delay</quote> feature adds one new
+          config variable,
+          <link linkend="pager-read-delay">$pager_read_delay</link>, which
+          is an integer for how many seconds the pager must remain on
+          a given message before marking it as read.  The variable
+          defaults to 0 for the default behavior of marking a message
```

Comment:
too many "default" in one line; would it read better as:
defaults to 0 for the previous behavior...

Reply:
yeah, that sounds a bit clearer